code/game/machinery/computer/robot.dm

```diff
@@ -8,98 +8,66 @@
 	icon_state = "robot"
 	req_access = list(access_robotics)
 	circuit = "/obj/item/weapon/circuitboard/robotics"
-
-	var/id = 0.0
-	var/temp = null
-	var/status = 0
-	var/timeleft = 60
-	var/stop = 0.0
-	var/screen = 0 // 0 - Main Menu, 1 - Cyborg Status, 2 - Kill 'em All! -- In text
-
 	light_color = LIGHT_COLOR_PINK
 
-
 /obj/machinery/computer/robotics/attack_ai(var/mob/user as mob)
 	src.add_hiddenprint(user)
 	return src.attack_hand(user)
 
 /obj/machinery/computer/robotics/attack_paw(var/mob/user as mob)
-
 	return src.attack_hand(user)
-	return
 
 /obj/machinery/computer/robotics/attack_hand(var/mob/user as mob)
 	if(..())
 		return
-	if (src.z > 6)
```

Comment:
Why remove this?

Reply:
there's no real need for this check if borgs can't have things done to them if they're in separate z-levels. I can't imagine a scenario where the robotics console would end up in z-level 7 and then end up being problematic for station borgs, also considering that this change prevents that too.